script/openqa-load-templates

```diff
@@ -104,53 +104,53 @@ sub print_error ($res) {
             }
         }
         die "ERROR: $err->{code} - $err->{message}" if $err->{code};
     }
     die "unknown error code - host $url->{host} unreachable?";
 }
 
-sub post_entry ($table, $entry) {
-    my %param;
-
-    if ($table eq 'JobGroups') {
-        # Create the group first
-        my $job_groups_url = $url->clone->path($options{apibase} . '/job_groups');
-        my $res = $client->post($job_groups_url, form => {name => $entry->{group_name}})->res;
-        print_error $res unless $res->is_success;
+sub handle_job_groups ($entry) {
+    # Create the group first
+    my $job_groups_url = $url->clone->path($options{apibase} . '/job_groups');
+    my $res = $client->post($job_groups_url, form => {name => $entry->{group_name}})->res;
+    print_error $res unless $res->is_success;
 
-        # Post the job template YAML
-        my $job_templates_url = $url->clone->path($options{apibase} . '/job_templates_scheduling');
-        $res
-          = $client->post($job_templates_url,
-            form => {name => $entry->{group_name}, template => $entry->{template}, schema => 'JobTemplates-01.yaml'})
-          ->res;
-        print_error $res unless $res->is_success;
-        return 1;
-    }
+    # Post the job template YAML
+    my $job_templates_url = $url->clone->path($options{apibase} . '/job_templates_scheduling');
+    $res
+    = $client->post($job_templates_url,
+        form => {name => $entry->{group_name}, template => $entry->{template}, schema => 'JobTemplates-01.yaml'})
+    ->res;
+    print_error $res unless $res->is_success;
+    return 1;
+}
 
-    if ($table eq 'JobTemplates') {
-        unless (defined($entry->{prio})) {
-            # we have to migrate the prio from the TestSuite to the JobTemplate
-            for my $ts (@{$info->{TestSuites}}) {
-                if ($ts->{name} eq $entry->{test_suite}{name}) {
-                    $entry->{prio} = $ts->{prio};
-                }
-            }
-        }
-        unless (defined($entry->{group_name})) {
-            # we have to create a group_name from the Product
-            my $gn = $entry->{product}{distri};
-            if ($entry->{product}{version} ne '*') {
-                $gn .= "-" . $entry->{product}{version};
-            }
-            $entry->{group_name} = $gn;
+sub handle_job_templates ($entry) {
+    unless (defined($entry->{prio})) {
+        # we have to migrate the prio from the TestSuite to the JobTemplate
+        for my $ts (@{$info->{TestSuites}}) {
+            $entry->{prio} = $ts->{prio} if $ts->{name} eq $entry->{test_suite}{name};
         }
     }
+    unless (defined($entry->{group_name})) {
+        # we have to create a group_name from the Product
+        my $gn = $entry->{product}{distri};
+        $gn .= "-" . $entry->{product}{version} if $entry->{product}{version} ne '*';
+        $entry->{group_name} = $gn;
+    }
+}
+
+sub handle_other_settings ($entry, $key) {
+}
+
```

Comment:
And this is a dead code (I believe it was intended to put the [for loop from line 155](https://github.com/os-autoinst/openQA/pull/6004/files#diff-2464c29d043a60435b7b92676d70ee13d004ef10e7c73a3352f27f3682a412b3L155) there?)
```suggestion

```

Reply:
yeah, you got me :) Sorry for the inferior quality PR.

Reply:
Wait, this wasn't a test? 😄

Reply:
you passed